### DISCUSSION TITLE
How to SELF Register a user with REST API

### DISCUSSION THREAD

**Discussion by YvannPONCE** [keycloak/keycloak#25040] [Q&A]
Hi, I have been looking to implement user registration in my own app without using the keycloack login page. I need an endpoint for the users to register themselves (without being admin) I can't find a working endpoint. It should be 

/realms/myrealm/protocol/openid-connect/registrations

He is waiting for a x-www-form-urlencoded body what keys/parameter I need to register

**Comment by iTrooz** [comment]
I'm not sure but I think that endpoint is made for use by browsers and is not really an API. (btw it was deprecated: https://github.com/keycloak/keycloak/pull/35903)
I don't think Keycloak supports this feature currently
